pkg/renderer/helm_values.go

```diff
@@ -191,6 +191,10 @@ customize:
     {{ range $key, $value := $envVars -}}
     {{ quote $key }}: {{ quote $value }}
     {{ end }}
+
+monitoring:
+  openshift:
+    enabled: {{ .K8sConfig.Monitoring.OpenShiftMonitoring }}
```

Comment:
how about cleaning this up and moving const to a file and `embed` it? this will be easier to read and edit in the future